config/routes.rb

```diff
@@ -4,14 +4,17 @@
   get "/games", to: 'homes#index'
   get "/games/new", to: 'homes#index'
   get "/games/:id", to: 'homes#index'
+  get "/games/:id/reviews/new", to: "home#index"
 
   devise_for :users
 
   namespace :api do
     namespace :v1 do
-      resources :games, only: [:index, :show, :create] do
-        resources :reviews, only: [:index]
-      end
+      resources :games, only: [:index, :show, :create]
+      resources :reviews, only: [:create]
     end
```

Comment:
you actually had it closer before. resources will still be nested under games, but will only have `:create`